### PR TITLE
Feature #151 sign up

### LINKED ISSUE #151: (SP: 1) Create a pop up window with sign up fields.
UserStory: #79 

## Description
The user clicks on the "Become a student" or "Become a tutor" buttons and Sign up form must pop up. 
## Design 
Image
Image
## Acceptance Criteria
- [ ] The pop-up window must contain all the elements in accordance with the mockup.
- [ ] The clicking elements must be disabled for now.

### PR DESCRIPTION
**Pull request by AndriiNasadiuk**
close #151 

(https://www.loom.com/share/dbec3633cd6049d88d26d2cf81b0983d?sid=8dfd0b4d-cb26-4375-a259-fa5a1380d86c)

### CODE REVIEW

**Review by Mygal-Alex — APPROVED** [review]
👍

**Review by marianamartsy — APPROVED** [review]
👍

**Review by ArtemHolikov — APPROVED** [review]
👍

**Review by LeonidShamarin — APPROVED** [review]

File: src/containers/guest-home-page/signup-form/SignupForm.jsx [hunk 1]
```diff
@@ -143,7 +143,13 @@ const SignupForm = ({
       </Box>
 
       <AppButton
-        disabled={!isAgreementChecked}
+        disabled={
+          !isAgreementChecked ||
+          !data.firstName ||
+          !data.lastName ||
+          t(errors.email) ||
+          t(errors.confirmPassword)
```
Comment (vlad-khrychov) [inline src/containers/guest-home-page/signup-form/SignupForm.jsx]: Why we need to call function `t` when we check only existing of errors, not its text?
```suggestion
          errors.email ||
          errors.confirmPassword
```

**Review by vlad-khrychov — COMMENTED** [review]
Will it work without `t`? If "no" you can merge PR like now.

File: src/containers/guest-home-page/signup-form/SignupForm.jsx (hunk 1 above)
Comment (AndriiNasadiuk) [reply src/containers/guest-home-page/signup-form/SignupForm.jsx]: You were right , i removed t() function and code still works . Also I have found one more validation bug, so I fixed it. I’m going to merge my branch

**Pull request merged**